### PR TITLE
Added angular directive that uses jquery-minicolor which is now in use by the band directive when choosing a color.

### PR DESCRIPTION
**Pull request by JacobBrandt**
screen shot 2016-11-30 at 5 59 50 pm

Resolves Issue: #11

### CODE REVIEW

File: public/directives/minicolors.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+// The MIT License (MIT)
```
Comment (nreese) [inline public/directives/minicolors.js]: All third party code should exist under the `public/lib` folder. Please move this file under lib/jquery.minicolors

File: public/directives/bands.html [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
-    <input
-      ng-model="band.color"
-      required
-      type="text"
-      class="form-control"
-      name="color"
-      style="width: 25%; background-color: {{band.color}};"
-    >
-    <input type="button" value="-" ng-click="removeBand()" class="btn btn-default btn-xs" ng-if="$index !== 0 && $last">
+    <div class="band">
```
Comment (nreese) [inline public/directives/bands.html]: Why did you add a new div element instead of just adding the class to the existing div element used by the ng-repeat?

Comment (JacobBrandt) [inline public/directives/bands.html]: no reason.  fixed
